src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -7308,59 +7308,67 @@ instruct minD_reg_reg(fRegD dst, fRegD src1, fRegD src2, rFlagsReg cr) %{
                  true /* is_double */, true /* is_min */);
   %}
 
-  ins_pipe(fp_dop_reg_reg_d);
+  ins_pipe(pipe_class_default);
 %}
 
 // Float.isInfinite
 instruct isIniniteF_reg_reg(iRegINoSp dst, fRegF src)
```

Comment:
Here is a typo and could you please fix it too?
s/isIniniteF_reg_reg/isInfiniteF_reg_reg

Reply:
> Here is a typo and could you please fix it too? s/isIniniteF_reg_reg/isInfiniteF_reg_reg

Thanks for your review, done.